Gemfile

```diff
@@ -4,29 +4,28 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.22'
-gem 'rack-cache', '< 1.3.0'
+gem 'rails', '4.1.5'
```

Comment:
`~> 4.1` or a specific version would be fine, yep.  develop works this way now, the version in the SCL is just an RPM packaging change, so for Bundler users it should be the current release to get security fixes.